src/types/select.js

```diff
@@ -25,7 +27,9 @@ export default  ngModule => {
           optionsAttr: check.string.optional,
           labelProp: check.string.optional,
           valueProp: check.string.optional,
-          groupProp: check.string.optional
+          groupProp: check.string.optional,
+          notNull: check.boolean.optional,
```

Comment:
You can correct it :) Feel free to [makeapullrequest.com](http://makeapullrequest.com) :)